### ISSUE TITLE
Clockdrones cannot use half of sigils in combat/critical situations cause are not working on spawn or something

### ISSUE THREAD

**Issue by Ratvar** [yogstation13/yogstation#2249]
Be clockdrone
Make sigil underneath stunned/critted/sleeping man
IT DOES NOT WORK UNTILL YOU PULL THEM OFF AND OVER IT
remember that u cannot pull cause u suck
 cry cause you cannot use sigils in half of situations-no reviving dead,dusting dead,converting stunned with ascension et cetera
 such a life of cockdrone

**Comment by ShadowDeath6** [comment]
>in combat

found your problem, they're base builders

**Comment by Ratvar** [comment]
No, this is problem with any sigil. You cannot make convert rune under security and just wait-you need to unpull him and pull him back

**Comment by Ratvar** [comment]
Those "basebuilders" can weld,spear and heal people fine. They are not just basemakers

**Comment by AdamElTablawy** [comment]
that seems to be because stepped() or whatever the proc is doesn't account for anything on top of the sigil when first created, only new mobs passing over

**Comment by Mrcreepssss** [comment]
couldnt you like make it so on creation the rune checks if it can activate? hmmm then you could get multiple conversions from the same rune with a rune you're not supposed to couldnt you?
make it so if that happens have RNGesus decides?

**Comment by ShadowDeath6** [comment]
having RNG decide things like stun and conversion are really bad practices

**Comment by Mrcreepssss** [comment]
well does it really matter? if you scribe it UNDERNEATH someone, or multiple then one of them would be stunned, the conversion one takes a bit and the other guy would drag them away. but it's alot more likely there's a bunch of other clockies that'll keep them there so yeah. does it really matter?
